packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -559,6 +534,8 @@ export default class TradeStore extends BaseStore {
                     }),
                 });
                 return;
+            } else if (!is_on_mf_account) {
```

Comment:
@matin-deriv we might not need the opposite condition, please check :)

```suggestion
            } else {
```

Reply:
Fair point, I have updated the code @maryia-deriv